app/uk/gov/hmrc/helptosavefrontend/connectors/EmailVerificationConnector.scala

```diff
@@ -43,7 +43,9 @@ trait EmailVerificationConnector {
 class EmailVerificationConnectorImpl @Inject() (http: WSHttp, conf: Configuration)(implicit crypto: Crypto)
   extends EmailVerificationConnector with ServicesConfig with Logging {
 
-  val linkTTLMinutes: Int = conf.underlying.getInt("microservice.services.email-verification.linkTTLMinutes")
+  override def runModeConfiguration: Configuration = conf
+
+  val linkTTLMinutes: Int = getInt("microservice.services.email-verification.linkTTLMinutes")
```

Comment:
why cant we move this into FrontendAppConfig ?

Reply:
this value only makes sense in this connector - there's nothing else in the project which needs this value